tree/30_generic_methods/kernel_module_loaded.cf

```diff
@@ -0,0 +1,71 @@
+#####################################################################################
+# Copyright 2019 Normation SAS
+#####################################################################################
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, Version 3.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+######################################################################################
+
+# @name Kernel module loaded
+# @description Ensure that a given kernel module is loaded on the system
+# @documentation Ensure that a given kernel module is loaded on the system.
+#   If a module is found not loaded, it will try to load it via modprobe.
```

Comment:
put it more simply: "If the module is not loaded, it will try to load it with modprobe"